cartography/cli.py

```diff
@@ -570,7 +570,7 @@ def run_azure(request):
         azure_azure_scope=request['azure']['azure_scope'],
         params=request['params'],
         neo4j_max_connection_lifetime=3600,
-        azure_requested_syncs=request['services']
+        azure_requested_syncs=request.get('services', None)
```

Comment:
same as gcp, we should use the default services list, if `services` parameter is null.

Reply:
for this we already have logic in __init.py__ file